cmd/appregistry-server/main.go

```diff
@@ -57,7 +57,7 @@ func runCmdFunc(cmd *cobra.Command, args []string) error {
 	}
 	err = log.AddDefaultWriterHooks(terminationLogPath)
 	if err != nil {
-		return err
+		logrus.WithError(err).Warn("unable to set termination log path")
```

Comment:
should this actually error out at all? won't it prevent you from running locally in a lot of cases?

Reply:
With these changes, this doesn't error out, it just prints a warning log on error.